customcomponent/src/components/Button.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const Button = ({ number }) => {
+  const handleClick = () => {
+    // eslint-disable-next-line
+    if (number == 1) {
+      alert("you clicked on button 1"); // eslint-disable-next-line
+    } else if (number == 2) {
+      alert("You clicked on button 2"); // eslint-disable-next-line
+    } else if (number == 3) {
+      alert("You clicked on button 3");
+    }
+  };
+
```

Comment:
You are using complex conditions that are not needed; I would suggest using this.

```suggestion
  
  const handleClick = (event) => {
    event.preventDefault();
    alert('You clicked on button' + number)
  };

```